### PR TITLE
SGSSB: Remove inactive auths + AB's Custom Ability

### PR DESCRIPTION
**Pull request by DeathlyPlays**
Vxn and I know the ability works lol we spent a few days on this

### CODE REVIEW

**Comment by ghost** [comment]
I completely <s>dis</s>agree with the above statement

File: mods/sgssb/abilities.js [hunk 1]
```diff
@@ -27,4 +27,24 @@ exports.BattleAbilities = {
 			this.setTerrain('');
 		},
 	},
+	conflictofinterest: {
+		id: "conflictofinterest",
+		name: "Conflict of Interest",
+		//Since levitate cant be coded in
```
Comment (HoeenCoder) [inline mods/sgssb/abilities.js:34]: Im sure it can, we just need to look tbh :p

Comment (ghost) [reply mods/sgssb/abilities.js:34]: true

Comment (DeathlyPlays) [reply mods/sgssb/abilities.js:34]: Levitate is ran from the battle-engine.js code e.e

Comment (HoeenCoder) [reply mods/sgssb/abilities.js:34]: isnt there a tag of some kind? or possibly a way to call that code?

Comment (ghost) [reply mods/sgssb/abilities.js:34]: Idk

Comment (ghost) [reply mods/sgssb/abilities.js:34]: OK THE THRILLING CONCLUSION! I did some digging and so I found out there is this thingy in `/sim/pokemon.js`

`isGrounded(negateImmunity)`


Maybe we can use `isGrounded: false,` for what we can do?

Comment (ghost) [reply mods/sgssb/abilities.js:34]: I think this is what you wanted @HoeenCoder

Comment (HoeenCoder) [reply mods/sgssb/abilities.js:34]: possibly, i need to research it.

Comment (DeathlyPlays) [reply mods/sgssb/abilities.js:34]: I could not say thats tested lol

Comment (ghost) [reply mods/sgssb/abilities.js:34]: We can also have an innate ability as well?

Comment (ghost) [reply mods/sgssb/abilities.js:34]: But idk

**Comment by ghost** [comment]
@HoeenCoder Thoughts on innate abilities? It could work and will add new depth to stuff

**Comment by HoeenCoder** [comment]
Youll need to tell me what you mean by innate abilities

**Comment by ghost** [comment]
Yeah nvm idk if it can be done

**Pull request merged**